src/experimentation-opportunities/experimentation-opportunities.js

```diff
@@ -167,6 +167,7 @@ async function updateRecommendations(oppty, context, site) {
   let lambdaResult;
   try {
     // eslint-disable-next-line no-await-in-loop
+    log.info('Invoking lambda function with payload: ', JSON.stringify(lambdaPayload, null, 2));
```

Comment:
please do not log this big payloads

Reply:
payload isn't big actually, its around 10 lines only